cmd/oidc-broker/main.go

```diff
@@ -20,7 +20,7 @@ import (
 
 func main() {
 	i18n.InitI18nDomain(consts.TEXTDOMAIN, po.Files)
-	a := daemon.New("")
+	a := daemon.New("oidc-broker")
```

Comment:
No, because we will not be able to rename it. This is why (see the diff, it’s `filepath.Base(os.Args[0])`) and not hardcoded.

Reply:
oops, you're right! I'll fix it.